### PR TITLE
Add LOGO url In QR Code API.

### PR DESCRIPTION
**Pull request by bucharitesh**
### GOAL 

Add functionality for QR Api to generate QR using Logo url in the center.

**FIXED:**

- [x] Add logo url in search param
- [x] Add checks if no logo url is passed
- [x] Add logo param in open api

### CODE REVIEW

**Comment by bucharitesh** [comment]
@steven-tey 👋 This PR is ready for code review

File: apps/web/lib/qr/utils.tsx
Comment (versecafe) [inline apps/web/lib/qr/utils.tsx:1]: <Image> from "next/image" instead?

**Comment by steven-tey** [comment]
Just seeing this – thanks a lot for your contribution @bucharitesh! Unfortunately the QR code doesn't seem to work with an image:

Screenshot 2024-02-15 at 7 02 25 PM

I believe this has something to do with how the `excavate` prop works? 👀

**Comment by bucharitesh** [comment]
It should have worked fine !!! Tried it a lot, can you check once again ? @steven-tey

**Comment by steven-tey** [comment]
Still doesn't work unfortunately 

image

**Pull request closed without merging**